### ISSUE TITLE
XAML Controls Gallery : Narrator announces the state as collapsed for home, Settings button in Navigation View even though it's not applicable.

### ISSUE THREAD

**Issue by StephenLPeters** [microsoft/microsoft-ui-xaml#2598]
Copied from internal [issue](https://microsoft.visualstudio.com/OS/_workitems/edit/26617757?src=WorkItemMention&src-action=artifact_link).

**Pre-requisite:** 
Install XAML CONTROLS Gallery from Store
Turn on Narrator.
Set narrator to item mode.

**Repro steps :**
Launch XAML Control Gallery app.
Activate "NavigationView" page from All Controls Page.
Navigate to "Hierarchical Navigation View ->Home 
Observe the narrator description .

**Expected result :**
 Narrator should announce proper description for home, Settings button in Navigation View.

**Actual result :**
 Narrator announces the state as collapsed for home, Settings button in Navigation View.
![Xaml_state](https://user-images.githubusercontent.com/7758786/83698659-db4b5680-a5b6-11ea-96bd-d060d0800558.png)

**Comment by StephenLPeters** [comment]
@ojhad for FYI
